package/MDAnalysis/coordinates/memory.py

```diff
@@ -502,8 +502,10 @@ def timeseries(self, asel=None, start=0, stop=-1, step=1, order='afc'):
             data is returned whenever `asel` is different from ``None``.
         start : int (optional)
         stop : int (optional)
+            .. deprecated:: 3.0
+                Note that `stop` is currently *inclusive* but will be
+                deprecated in favour of being *exclusive* in 3.0.  
         step : int (optional)
-            range of trajectory to access, `start` and `stop` are *inclusive*
```

Comment:
Might be missing something but diff indicates no replacement for this? You probably should have done kind of "this is what step does"?